doc/releases/release-notes-4.0.rst

```diff
@@ -163,6 +163,15 @@ Drivers and Sensors
 
 * GPIO
 
+  * pca_series: added ``pca_series`` driver to unify support to tca and pca series I2C-based
```

Comment:
Move these changes to the 4.1 release notes since we missed the 4.0 feature freeze